### ISSUE TITLE
Getting Started improvements

### ISSUE THREAD

**Issue by websocket98765** [Dan6erbond/sk-auth#93]
Hi, I see a lot of effort has gone into this, but I'm not able to get it to work--the Getting Started assumes too much existing knowledge for me and I'm missing pieces; the example code seems to also be built with older versions of Svelte pieces or I'm missing something, given it's a full repo and hard to know if I've done everything needed.

Can I kindly request [Getting Started](https://github.com/Dan6erbond/sk-auth#getting-started) section to have Steps 1,2,3...; file names and full contents of each file please? I'd love to use this!

Thanks!

**Comment by moreorover** [comment]
As for myself I found [this blog post](https://robertohuertas.com/2021/07/25/sveltekit-cognito-authentication/) very helpful.

**Comment by VoiceOfSoftware** [comment]
Agreed, I have no idea how to get started. There's a sample app inside the "app" directory, but it doesn't build. And the instructions say it's easy to add to your own app, but do not say where to put that code, nor how to import the libraries needed to make the code compile.

And there seem to be a ton of dependencies on things like tailwindcss, something called "clsx", and other frameworks I've never heard of. I mean, yes, I've heard of tailwind, but can't seem to get past the compile errors.

I would *really* like to love this project, because auth is hard to do, but I am resorting to rolling my own.
